docs/guides/debugging.md

```diff
@@ -149,6 +149,30 @@ places. You can always load it by hand if GDB refuses or fails to load it:
 
     (gdb) source /path/to/your/.gdbinit
 
+Here's an example of a `.gdbinit` file helpful for debugging scylla.
+Note that it registers pretty-printer helpers for printing common
+std-c++ stl containers:
+```
+handle SIG34 pass noprint
+handle SIGUSR1 pass noprint
+set print pretty
+set print static-members no
+set python print-stack full
+set auto-load safe-path /opt/scylladb/libreloc
+add-auto-load-safe-path /lib64
+add-auto-load-safe-path /usr/lib64
+set solib-search-path /opt/scylladb/libreloc:/lib64:/usr/lib64
+set pagination off
```

Comment:
I wouldn't disable pagination by default. Disabling it is situationally useful, but pagination on is a better default.

Reply:
I find it distracting and problematic when copy-pasting into github issues

Reply:
I'll remove it for now.
We can debate it later, after this PR is merged.
It's not that urgent :)

Reply:
I personally find paging useful and only disable when logging into a file.
I think such personal preferences should be kept out of a "baseline" gdbinit and we should rely on gdb defaults instead.